app/views/exams/StartPage.vue

```diff
@@ -72,7 +74,7 @@ export default {
       return this.exam?.languages || ['python', 'javascript']
     },
     hasPermission () {
-      const clans = me.get('clans')
+      const clans = me.get('clans') || []
```

Comment:
i think the exam could be a feature that supports more customers. so I add the clan as permission checking . but we can hack it for MTO this time anyway. (also for testing you need to hack it to true)

Reply:
I'm ok with clan as long as we can test and make sure it will work for MTO otherwise student + isMto check is better. I did override it to true for testing.